src/server/components/dashboard/listsItems/item/update/actionHandlers/archive.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response } from "express";
+import { archiveListItem } from "server/models/listItem";
+
+export async function archive(req: Request, res: Response): Promise<void> {
+  const userId = req?.user?.userData?.id;
```

Comment:
I pulled out all the jsonData properties so they're on the class instead of nested. req.user.id should exist, see `src/server/components/auth/authenticated-user.ts`.

req shouldn't be nullish (so no coalesce after req)

Reply:
`req shouldn't be nullish (so no coalesce after req)` lol, such a schoolboy error 🤦🏿